advanced/Scripts/webpage.sh

```diff
@@ -238,18 +238,18 @@ trust-anchor=.,20326,8,2,E06D44B80B8F1D39A95C0B0D7C65D08458E880409BBC68345710423
         #          168.192.in-addr.arpa to 192.168.0.0/16
         #          192.in-addr.arpa to 192.0.0.0/8
         if [[ "${CONDITIONAL_FORWARDING_REVERSE}" == *"in-addr.arpa" ]];then
-            arrRev=("${CONDITIONAL_FORWARDING_REVERSE//./ }")        
-            case ${#arrRev[@]} in 
+            arrRev=("${CONDITIONAL_FORWARDING_REVERSE//./ }")
```

Comment:
Thanks, I've been meaning to PR this whitespace away since I [introduced it](https://github.com/pi-hole/pi-hole/commit/e004d27b28bf3da43101948903c56a7cf66be4eb) 😉

Reply:
I did not intend to change the whitespaces and was surprised they appear to be removed here. Must be a default setting in `atom` editor.